src/sagemaker/serve/utils/telemetry_logger.py

```diff
@@ -63,13 +66,20 @@ def wrapper(self, *args, **kwargs):
                 f"{func_name}"
                 f"&x-modelServer={MODEL_SERVER_TO_CODE[str(self.model_server)]}"
                 f"&x-imageTag={image_uri_tail}"
+                f"&x-sdkVersion={SDK_VERSION}"
             )
 
             if self.model_server == ModelServer.DJL_SERVING or self.model_server == ModelServer.TGI:
                 extra += f"&x-modelName={self.model}"
 
+            if self.sagemaker_session.endpoint:
+                extra += f"&x-endpointArn={self.sagemaker_session.endpoint['EndpointArn']}"
+
+            start_timer = perf_counter()
```

Comment:
What unit/format is this?

Reply:
Second -> https://docs.python.org/3/library/time.html#time.perf_counter:~:text=in%20version%203.7.-,time.perf_counter(),-%E2%86%92%20float%C2%B6

Reply:
Could we cap the fractional part to hundredths or something if it's not capped? E.g. 12.34 instead of 12.3419485759.